packages/react-filerobot-image-editor/src/components/common/AnnotationOptions/index.jsx

```
@@ -52,19 +53,25 @@ const AnnotationOptions = ({
       },
-      ...(!useCloudimage
+      ...(!useCloudimage && hideStrokeField
         ? [
             { titleKey: 'stroke', name: POPPABLE_OPTIONS.STROKE, Icon: Stroke },
             { titleKey: 'shadow', name: POPPABLE_OPTIONS.SHADOW, Icon: Shadow },
           ]
+        : !useCloudimage
+        ? [
+            { titleKey: 'shadow', name: POPPABLE_OPTIONS.SHADOW, Icon: Shadow },
+          ]
+        : []),
```

ESLint

[error] Do not nest ternary expressions.

(no-nested-ternary)